api_review/RadialGradientBrush/RadialGradientBrush.md

```diff
@@ -110,15 +111,15 @@ This example creates a radial gradient that uses Absolute mapping mode with cust
 
 ## Gradient layout
 
-The gradient is drawn within an ellipse that is defined by the `Center` and `EllipseRadius` properties. Colors for the gradient start at the center of the ellipse and end at the radius.
+The gradient is drawn within an ellipse that is defined by the `Center`, `RadiusX`, and `RadiusY` properties. Colors for the gradient start at the center of the ellipse and end at the radius.
 
 The colors for the radial gradient are defined by color stops added to the `GradientStops` collection property. Each gradient stop specifies a color and an offset along the gradient.
 
 The gradient origin defaults to center and can be offset using the `GradientOrigin` property.
 
-`MappingMode` defines whether `Center`, `EllipseRadius` and `GradientOrigin` represent relative or absolute coordinates.
+`MappingMode` defines whether `Center`, `RadiusX`, `RadiusY`, and `GradientOrigin` represent relative or absolute coordinates.
 
-When `MappingMode` is set to `RelativeToBoundingBox`, the X and Y values of the three properties are treated as relative to the brush's rendered bounds, where `(0,0)` represents the top left and `(1,1)` represents the bottom right of the brush's rendered bounds for the `Center` and `EllipseRadius` properties and `(0,0)` represents the center for the `GradientOrigin` property.
```

Comment:
We should say the element bounds instead of the brush's rendered bounds. That is the current behavior (same as WPF). We say it here and a couple more times in the document.

Reply:
Thanks, fixed!